### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by janis-jenny**
- [ ]  Install webpack plugins and loaders from a template
- [ ]  Create 3 Classes: ApiData, Render and Form
- [ ]  Use DOM to render the data of a city.

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Invalid :red_circle: 
The project should have a descriptive README.md. The title of the project and the project description description don't correspond to your APP.
Please use this file as linter config : [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml)

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,8 @@
...
-const displayJoke = async () => {
-  const joke = await requestJoke();
-  container.innerText = joke.value
-}
-
-displayJoke()
\ No newline at end of file
+form.createForm();
+form.submitForm();
+// console.log(process.env.DB_HOST);
```
Comment (umairarshadbutt) [inline src/index.js]: remove commented code

File: src/components/background.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import clouds from '../images/clouds.gif';
+import clear from '../images/clear_sky.gif';
+import rain from '../images/rain.gif';
+import thunder from '../images/thunder.gif';
+import snow from '../images/snow.gif';
+import mist from '../images/mist.gif';
+/* eslint-enable  no-unused-vars */
+
+const renderBackground = (data) => {
+  console.log(data.weather.toLowerCase());
```
Comment (umairarshadbutt) [inline src/components/background.js]: Remove `console` statement

File: src/components/form.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    render.renderDegrees();
+    render.renderTemp();
+    render.renderHumidity();
+    render.renderClimate();
+    render.renderWind();
+    render.renderCountry();
+    renderBackground(value);
+    console.log('HERE');
+    console.log(value);
```
Comment (umairarshadbutt) [inline src/components/form.js]: Remove `console` statements

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Such a remarkable project you made :+1: ! Awesome README :100: !
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested changes  👇.
- [ ] Remove commented code and console from JavaScript file.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] Live link is not found in the readme file.
- [ ] Please add more instructions about the `.env` file in `How it works` README file section. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] API response is giving the 401 status in networks and in dom it is showing 404 status

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad (Code Reviewer)](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: src/components/api.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const BASE_URL = 'https://api.openweathermap.org/data/2.5/weather?';
+/* eslint-disable  prefer-destructuring */
+const API_KEY = process.env.API_KEY;
+/* eslint-enable  prefer-destructuring */
+export { BASE_URL, API_KEY };
```
Comment (Temmarie) [inline src/components/api.js:5]: - [ ] _[OPTIONAL]_ Inline comment:

With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

File: src/components/form.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    const form = document.getElementById('from-city');
+    form.addEventListener('submit', (e) => {
+      e.preventDefault();
+      const cityInput = form.elements.city;
+      Form.weather(cityInput);
+      cityInput.value = '';
+    });
+  }
+  /* eslint-enable  class-methods-use-this */
+}
```
Comment (Temmarie) [inline src/components/form.js:59]: - [ ] _[OPTIONAL]_ Consider refactoring your code to fix the underlying eslint issues instead of disabling it.

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @janis-jenny  ! :wave:

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:
 
#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**